components/date-time/date.js

```diff
@@ -3,14 +3,15 @@
  */
 import ReactDatePicker from 'react-datepicker';
 import moment from 'moment';
+import { wpmoment } from '@wordpress/date';
 
 /**
  * Module Constants
  */
 const TIMEZONELESS_FORMAT = 'YYYY-MM-DDTHH:mm:ss';
 
 function DatePicker( { currentDate, onChange, ...args } ) {
-	const momentDate = currentDate ? moment( currentDate ) : moment();
+	const momentDate = currentDate ? moment( currentDate ) : wpmoment;
```

Comment:
Should this be `wpmoment()`?

Also why we're using `moment` sometimes and sometimes `wpmoment`  can we use `wpmoment` consistently.

Reply:
I think this was just an oversight, we should be able to use wpmoment everywhere. Also, that probably should be wpmoment() - I'll search and double check them all again and any other instances.